setup-nexus-npm/action.yml

```diff
@@ -0,0 +1,19 @@
+name: setup-nexus-npm
+inputs:
+  endpoint:
+    required: true
+  login:
+    required: true
+  password:
+    required: true
+runs:
+  using: composite
+  steps:
+    - run: |
+        rm -f ~/.npmrc
```

Comment:
I don't think we need to delete the file if we're going to truncate it before write, anyway